packages/editor/src/components/post-publish-panel/maybe-category-panel.js

```diff
@@ -14,6 +14,20 @@ import HierarchicalTermSelector from '../post-taxonomies/hierarchical-term-selec
 import { store as editorStore } from '../../store';
 
 function MaybeCategoryPanel() {
+	const hasSiteCategories = useSelect( ( select ) => {
+		const { getEntityRecords } = select( coreStore );
+		const categories =
+			getEntityRecords( 'taxonomy', 'category', { per_page: -1 } ) || [];
```

Comment:
I am not sure `per_page: -1` is correct here, we only need to know if there is more than the default category 🤔

Reply:
Right, this over fetches the results. There's some room to optimize the queries and make them conditional. Maybe use the [`exclude` argument](https://developer.wordpress.org/rest-api/reference/categories/#arguments) to fetch only a single category.

Reply:
To exclude the default category id, we would need to look that ID up first.